packages/android_alarm_manager/lib/android_alarm_manager.dart

```diff
@@ -43,10 +43,7 @@ void _alarmManagerCallbackDispatcher() {
 
   // Once we've finished initializing, let the native portion of the plugin
   // know that it can start scheduling alarms.
-  // TODO(amirh): remove this on when the invokeMethod update makes it to stable Flutter.
-  // https://github.com/flutter/flutter/issues/26431
-  // ignore: strong_mode_implicit_dynamic_method
-  _channel.invokeMethod('AlarmService.initialized');
+  _channel.invokeMethod<void>('AlarmService.initialized');
```

Comment:
This should be bool following the fix in https://github.com/flutter/plugins/pull/1384